worker/uniter/runner/jujuc/context.go

```diff
@@ -30,6 +31,10 @@ type Context interface {
 	actionHookContext
 	unitCharmStateContext
 	workloadHookContext
+
+	// GetLogger returns a juju loggo Logger for the supplied module that is
+	// correctly wired up for the given context
+	GetLogger(module string) loggo.Logger
```

Comment:
There's already a GetLogger() on HookContext. I think that does the right thing. So we should just use that and not have to override it here. Or is the embedded HookContext one not suitable?